### DISCUSSION TITLE
분석 할 pr 개수

### DISCUSSION THREAD

**Discussion by jin7969** [woowacourse-study/2022-code-review-study-2#26] [General]
솔직히 다른게 할게 많아서 비중이 크진않다.. 하루 이틀정도??  나만그런가..ㅜ
3인페어를 하면 개숫가 11개인데.. 전부다 제대로 보기 쉽지는 않다..
분석할 개수가 딱 정해지니깐 약간 압박이 있는듯하다. 물론 장점도 있다!

그냥 생각 하기로는..
1. 분석 할 개수를 줄인다
2. 자유롭게 한다. 1개를 제대로 하든 37개 다 하든 ㅋㅋ
3. 지금이 좋타
4. 각자 리뷰어가 되어본다. 저희 4명 코드를 서로 씹어본다 ㅋ

✅ 목적이 아 내일 스터디인데 숙제 해야 되네... 아 할거 많은데... 가 아니라
     내일 스터디때 또 어떤 의견들이 나올까? 궁금한데!! 스터디가 기다려지는? 이런 느낌으로 유지 되었으면 좋겠다!

**Comment by euijinkk** [comment]
8명이니까 코드 자체를 보는 것이 아니라, 코드리뷰 내용만 보게 된다. 
코드를 제대로 알지 못 한채로 코드리뷰를 공부하려고 하니 피상적으로 이해하게 된다.

1. 1~2 명의 코드를 제대로 이해하고, 피드백을 정리하기

**Reply by euijinkk** [reply]
하지만, 몇개의 PR을 보아야 어떤 사람의 코드를 분석할지 결정할 수 있으므로, PR 분류는 해야할 것

**Comment by al-bur** [comment]
모든 PR의 코드들을 세부적으로 분석하는 것은 불가능하다고 생각했어요 🎃
애초에 스터디를 모집했을때 `전체 PR의 리뷰어님들의 리뷰`를 훑어볼 수 있도록 하는 것이 목적이었기 때문에 지금 방식이 괜찮아보이긴 합니다

만약, 양이 부담이 된다고 한다면, 서로 이 부분에 대해서 이야기를 해봐야할 것 같긴하네요!
